### PR TITLE
Add Passive Perception Floor for Perception Roll

### PR DESCRIPTION
**Pull request by Aeristoka**
Advanced Option

### CODE REVIEW

**Comment by Aeristoka** [comment]
Possible to also add Passive Investigation and Passive Insight to this as well, as 2x more advanced options.

File: src/common/roll_renderer.js [hunk 1]
```diff
@@ -435,6 +435,17 @@ class Beyond20RollRenderer {
...
+            let passive_perception = parseInt(character._passive_perception);
+            if (request.advantage == RollType.ADVANTAGE)
+                passive_perception += 5;
+            if (request.advantage == RollType.DISADVANTAGE)
+                passive_perception -= 5;
+            for (let i = 0; i < rolls.length; i++) {
+                if (rolls[i].total < passive_perception)
+                    rolls[i]._total = passive_perception;
+            }
+        }
```
Comment (kakaroto) [inline src/common/roll_renderer.js:448]: this would have had to be a `min10` modifier to the roll instead

**Comment by kakaroto** [comment]
Unsure about this, as far as I know, that's not really how it works.. or at least, there's a lot of debate on whether or not the passive is the floor.
Technically, you have a passive perception, passive insight, passive stealth, etc... and it's just 10+modifier. And considering that reliable talent and silver tongue are specific class features that makes your roll be a `min10` for specific skills, that means that class feature gives you your passive as the floor... so unless you have that as an actual class features, it wouldn't make sense that perception rolls would default to your passive as the floor, even as an advanced option, it seems to go against RAW and RAI.

**Comment by Aeristoka** [comment]
> Unsure about this, as far as I know, that's not really how it works.. or at least, there's a lot of debate on whether or not the passive is the floor. Technically, you have a passive perception, passive insight, passive stealth, etc... and it's just 10+modifier. And considering that reliable talent and silver tongue are specific class features that makes your roll be a `min10` for specific skills, that means that class feature gives you your passive as the floor... so unless you have that as an actual class features, it wouldn't make sense that perception rolls would default to your passive as the floor, even as an advanced option, it seems to go against RAW and RAI.

https://rpg.stackexchange.com/questions/48281/does-passive-perception-score-supersede-the-result-of-a-lower-active-perception Is the reference, thought I had included that here.

**Pull request closed without merging**